### PR TITLE
Use redux more

### PR DESCRIPTION
**Pull request by smcmurtry**
Trying to get tests working. I've only found hacky solutions so far.

### CODE REVIEW

**Comment by sastels** [comment]
resolves #441 I assume?

**Comment by smcmurtry** [comment]
Yes, this resolves #441

**Comment by sastels** [comment]
This isn't optimal
image

File: __tests__/components/BB_test.js [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 import { mount, shallow } from "enzyme";
 import React from "react";
+import configureStore from "redux-mock-store";
 
 import { BB } from "../../components/BB";
 import benefitsFixture from "../fixtures/benefits";
-import elegibilityPathsFixture from "../fixtures/eligibilityPaths";
+import eligibilityPathsFixture from "../fixtures/eligibilityPaths";
```
Comment (sastels) [inline __tests__/components/BB_test.js:9]: whoops

Comment (smcmurtry) [reply __tests__/components/BB_test.js:9]: yeah it took me a while to track down that bug...

**Review by sastels — APPROVED** [review]
LGTM! Great work!

**Review by maxneuvians — APPROVED** [review]
LGTM 🦑

**Pull request merged**